data/102/063/515/102063515.geojson

```diff
@@ -10,6 +10,12 @@
     "geom:latitude":49.560696,
     "geom:longitude":9.725084,
     "iso:country":"DE",
+    "label:deu_x_preferred_longname":[
+        "Landkreis Main-Tauber-Kreis"
```

Comment:
Another question about names, since this ends in `-Kreis` is the prefix `Landkreis ` necessary? 

- https://de.wikipedia.org/wiki/Main-Tauber-Kreis

Reply:
Seems likely, but please confirm.

Reply:
Ditto on the crawl to check "kreis" and "stadt" in the wof:name. Should fix this case, too.